app/controllers/ems_container_controller.rb

```diff
@@ -32,6 +32,11 @@ def ems_container_form_fields
     ems_form_fields
   end
 
+  def new
+    super
+    @ems_types -= [["Atomic", "atomic"], ["Atomic Enterprise", "atomic_enterprise"]]
```

Comment:
this removes atomic and atomic_enterprise from the dropdown in the "new container manager" dialog.
IIUC this is what you wanted, right?